CRM/Contribute/Import/Parser/Contribution.php

```diff
@@ -626,19 +615,14 @@ private function deprecatedFormatParams($params, &$values, $create = FALSE) {
 
       switch ($key) {
         case 'contact_id':
-          if (!CRM_Utils_Rule::integer($value)) {
-            return civicrm_api3_create_error("contact_id not valid: $value");
-          }
           $dao = new CRM_Core_DAO();
           $qParams = [];
-          $svq = $dao->singleValueQuery("SELECT is_deleted FROM civicrm_contact WHERE id = $value",
-            $qParams
-          );
+          $svq = CRM_Core_DAO::singleValueQuery('SELECT is_deleted FROM civicrm_contact WHERE id = ' . (int) $value);
```

Comment:
```suggestion
          $svq = CRM_Core_DAO::singleValueQuery('SELECT is_deleted FROM civicrm_contact WHERE id = %1', [1 => [$value, 'Integer']);
```

Reply:
Sure - it will be removed again soon anyway & they are much of a muchness to me